scilpy/io/streamlines.py

```diff
@@ -144,3 +144,33 @@ def load_tractogram_with_reference(parser, args, filepath,
         parser.error('{} is an unsupported file format'.format(filepath))
 
     return sft
+
+
+def filter_tractogram_data(tractogram, streamline_ids):
+    """ Filter tractogram data (data_per_streamline, data_per_point)
+    according to streamline ids
+
+    Parameters:
+    -----------
+    tractogram: Tractogram or StatefulTractogram
```

Comment:
Would be nice to support sft and tractogram. A simple check with isinstance would let you support both

Reply:
Looking at the code, I think both are already handled ? Both `nibabel.streamlines.tractogram.Tractogram` and `dipy.io.stateful_tractogram.StatefulTractogram` expose `data_per_streamline` and `data_per_point`, no ?